robotiq_description/urdf/robotiq_gripper.urdf.xacro

```diff
@@ -7,7 +7,7 @@
     <xacro:include filename="$(find robotiq_description)/urdf/robotiq_gripper_macro.urdf.xacro" />
 
     <link name="world" />
-    <xacro:robotiq_gripper parent="world" sim="$(arg use_sim)" prefix="">
+    <xacro:robotiq_gripper name="RobotiqGripperSystem" prefix="" parent="world" use_fake_hardware="$(arg use_sim)">
```

Comment:
Should we call this argument `use_fake_hardware` instead of `use_sim` to be consistent?